english/I'm-batman/not-batman.md

```diff
@@ -0,0 +1 @@
+Are you batman? Let's [find out](http://www.somethingawful.com/news/concealed-carry-renfest/1/)
```

Comment:
It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.